components/modal/modal.css

```diff
@@ -0,0 +1,39 @@
+@import url(/stylesheets/default.css);
+
+main header {
+  height: 100%;
+}
+.modal-link {
+  background-color: var(--extra-color);
+}
+
+/* modal css */
+.modal {
+  width: 20rem;
+  display: flex;
+  flex-direction: column;
+  padding: var(--gap-small);
+  margin: var(--gap-medium);
+  background-color: var(--secondary-color);
+}
+
+.header-modal {
+  display: flex;
+  justify-content: space-between;
+}
+
+.close {
+  background-color: var(--extra-color);
+  padding: var(--gap-smallest);
+  border-radius: 50%;
```

Comment:
Can create variables for border-radius instead of %, like `lightrounded : 4px` and `rounded: 999px;`

Reply:
i had created a class for this don't know how I forgot to use it. thank you ill do this.